usaspending_api/common/tests/test_endpoints.py

```diff
@@ -102,12 +102,13 @@ def evaluateEquivalence(item1, item2):
                 continue
             if 'req' in key:  # Ignore checksums for comparisons
                 continue
-            if 'next' in key:  # Ignore this due to checksum comparison
-                continue
-            if 'previous' in key:  # Ignore this due to checksum comparison
+            if 'page_metadata' in key:  # Ignore this due to checksum comparison
                 continue
             equality = equality and evaluateEquivalence(item1[key], item2[key])
     else:
         # Converting to string represenations to clear up some issues with Decimal casting
+        # print(str(item1))  # Leaving these here for future debugging
```

Comment:
As I said in the comment, these lines are super useful when testing why a check failed; so until we properly parameterize this I'm leaving them in here commented.